src/index.html

```diff
@@ -22,6 +22,486 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #header region -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__link-item">
+            <a
+              class="nav__link nav__link--is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="smartphones"
```

Comment:
According to the checklist, all links should have a leading dot to prevent potential issues with page navigation (e.g., href="./smartphones").